payment-failure/src/main/scala/com/gu/identity/paymentfailure/BrazeClient.scala

```diff
@@ -1,15 +1,29 @@
 package com.gu.identity.paymentfailure
 
 import com.typesafe.scalalogging.StrictLogging
+import scalaj.http.Http
+import io.circe.syntax._
+
 
 class BrazeClient extends StrictLogging{
 
-  def sendEmail(emailData: IdentityBrazeEmailData, emailToken: String) : Either[Throwable, BrazeResponse] = {
-    val isSuccess = true
-    logger.info(s"Sending payment failure request to braze for email ${emailData.emailAddress} with token $emailToken")
-    isSuccess match {
-      case true => Right(BrazeResponse("example success response"))
-      case false => Left(new Exception( s"Failed to send email from Braze"))
+  def sendEmail(emailData: IdentityBrazeEmailData, emailToken: String, config: Config) : Either[Throwable, BrazeResponse] = {
+    
+    logger.info(s"send BrazeEmail for email ${emailData.emailAddress} with token $emailToken with templateId ${emailData.templateId}")
+
+    val sendRequest = BrazeSendRequest(emailData.externalId, config.brazeApiKey, emailData.templateId, emailData.customFields + ("emailToken" -> emailToken))
+
+    val postResponse = Http(s"${config.brazeApiHost}/campaigns/trigger/send")
```

Comment:
Can this throw an exception? Should we handle it?

Reply:
Good shout